app/models/incomeSourceDetails/PropertiesRentedModel.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.incomeSourceDetails
+
+import models.core.CustomReads
+import play.api.libs.functional.syntax._
+import play.api.libs.json.{Json, Reads, _}
+
+case class PropertiesRentedModel(uk: Option[Int], eea: Option[Int], nonEea: Option[Int], total: Option[Int])
```

Comment:
Don't need to capture this

Reply:
I think we need this model as `IncomeSourceDetailsModel`depends on it. This is required for the class and tests. 
https://github.com/hmrc/income-tax-calculation/pull/45/files#diff-45ea0e11f0ba84afd3f55cffcdc56b5fc24edb6a642852da6a2d4d6d3fa89493R100